data/iAnonymous3000.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Sooraj Sathyanarayanan",
+  "type": "personal",
+  "bio": "Security & Privacy Researcher, International Chess Player♟️"
```

Comment:
Please add a comma at the end of your bio

`"bio": "Security & Privacy Researcher, International Chess Player♟️",`